src/libs/ajax.js

```diff
@@ -1130,6 +1130,21 @@ const AzureStorage = signal => ({
         storageContainerName: container.resourceAttributes.azureStorageContainer.storageContainerName
       }
     }
+  },
+
+  resources: async workspaceId => {
```

Comment:
Can you also call the new resources function up at line 1110?

Reply:
I've refactored to centralize the calls into your details fn, take a look.